04-mpu6050/scripts/mpu6050.sh

```diff
@@ -0,0 +1,109 @@
+#! /bin/bash
+
+ON=0x00
+OFF=0x40
+POWER=0xeb
+DEVICE=0x68
+DEVICE_ID=0
+
+help()
+{
+        printf "Usage: $0 [OPTION] ...\n"
+        printf "Script to help get the temperature from the mpu6050 device.\n\n"
+        printf "List of options:\n"
+        printf "  -h, --help\t\tdisplays help page.\n"
+        printf "  -i, --init\t\tinitializes the device.\n"
+        printf "  -d, --deactivate\tdeactivate the device\n"
+        printf "  -s, --show TYPE\tdisplays the temperature in the specified TYPE on the screen.\n"
+        printf "      --stream_data\tdisplays temperature data every second.\n"
+
+        printf "\nThe TYPE argument can take values that indicate in what form the temperature is displayed.
+This argument can only accept values: --show f (output in Fahrenheit) and --show c (output in Celsius).\n"
+
+        printf "\nFor the correct execution of the script, you may need super user rights. Also MUST be installed
+package і2с-tools.\n"
+
+        printf "\nAuthor: Maxim Primerov. Email: <primerovmax@gmail.com>. License: GNUv3.\n"
+}
+
+init()
+{
+        sudo i2cset -y ${DEVICE_ID} ${DEVICE} ${POWER} ${ON}
+}
+
+deactivate()
+{
+        sudo i2cset -y ${DEVICE_ID} ${DEVICE} ${POWER} ${OFF}
+}
+
+show()
+{
+        MAJOR=$(sudo i2cget -y 0 0x68 0x41)
+        MINOR=$(sudo i2cget -y 0 0x68 0x42)
+
+        SING=$(( (MAJOR & 0x80) >> 7))
+        TEMPERATURE=$(( (MAJOR << 8) + MINOR ))
+
+        if [[ ${SIGN} != 1 ]]
```

Comment:
Is this conversation correct? Did You measure temperature below and above 36.5 degrees C